Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -783,17 +820,28 @@ protected override void ProcessRecord()
                 contentStream = new MemoryStream(contentBuffer);
             }
 
+            // Get the existing storage object so we can use its metadata. (If it does not exist, we will fall back to
+            // default values.)
+            Object existingGcsObject = null;
+            Dictionary<string, string> existingObjectMetadata = null;
```

Comment:
It's a fair point, the "= null;" is not only optional, it is 100% redundant. But so is the semicolon (generally) in JavaScript.

But I've written enough C++ to never trusted an uninitialized variable. You might say I am broken, and need to be more trusting of the C# runtime. But I've seen things, man.